.env.example

```diff
@@ -6,11 +6,12 @@ TRANSACTIONS_CLOSE_PAYMENT_QUEUE_NAME=pagopa-ecommerce-transactions-close-paymen
 TRANSACTIONS_NOTIFICATIONS_RETRY_QUEUE_NAME=pagopa-ecommerce-transactions-notifications-retry-queue
 TRANSACTIONS_NOTIFICATIONS_QUEUE_NAME=pagopa-ecommerce-transactions-notifications-queue
 TRANSACTIONS_DEAD_LETTER_QUEUE_NAME=pagopa-ecommerce-transactions-dead-letter-queue
-QUEUE_CONNECTION_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://storage:10001/devstoreaccount1
+QUEUE_TRANSIENT_CONNECTION_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://storage:10001/devstoreaccount1
+QUEUE_DEADLETTER_CONNECTION_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount3;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://storage:10001/devstoreaccount1
 
-ECOMMERCE_STORAGE_QUEUE_ACCOUNT_NAME=devstoreaccount1
-ECOMMERCE_STORAGE_QUEUE_KEY=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
-ECOMMERCE_STORAGE_QUEUE_ENDPOINT=http://127.0.0.1:10001/devstoreaccount1
+ECOMMERCE_STORAGE_TRANSIENT_QUEUE_ACCOUNT_NAME=devstoreaccount1
+ECOMMERCE_STORAGE_TRANSIENT_QUEUE_KEY=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
+ECOMMERCE_STORAGE_TRANSIENT_QUEUE_ENDPOINT=http://127.0.0.1:10001/devstoreaccount1
```

Comment:
Are these used anywhere? Aren't thery redundant with `QUEUE_TRANSIENT_CONNECTION_STRING`?

Reply:
They are needed in application properties and are used by StorageQueueTemplate class used in the QueueConsumer class. Since the scope of this PR is to refers to new resources, I avoid to refactor this group of properties. It can be done in a dedicated PR, to not add more entropy than already present